recipes/ogre/1.x/test_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.definitions["OGRE_VERSION"] = tools.Version(self.deps_cpp_info["ogre"].version)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if tools.cross_building(self.settings):
```

Comment:
```suggestion
        if tools.cross_building(self):
```
self.settings is outdated for this method.